src/redux/ConfigureStore.js

```diff
@@ -0,0 +1,10 @@
+import { applyMiddleware, createStore } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import fetchMissions, { missionReducer } from './missions/missions';
+
+const store = createStore(missionReducer, applyMiddleware(logger, thunk));
+
+store.dispatch(fetchMissions());
+
```

Comment:
Great work so far 👏 👏 however, I have an observation here. Based on the requirements, please see image:

<img width="1304" alt="Screenshot 2022-02-28 at 20 51 36" src="https://user-images.githubusercontent.com/82473943/156049332-a7e1ecdf-a7e5-44ca-9cdd-b213647ca1fa.png">

The rockets component is the default view and therefore data should be fetched on the application's first run, however, for the mission's view **"the missions data should only be fetched (once) when a user navigates to the Missions section."** but based on the above code, missions data is fetched immediately store is initialized

Here are a few suggestions:
- Consider using the `useEffect` hook to fetch data when the user clicks to open the missions component
- The `useEffect` hook should not be passed an empty array dependency, rather the updated `missionState` array of objects can be passed as a dependency list `useEffect(() => {  use redux-thunk to fetch data }, [...state])` through a variable.

Reply:
Okay, I will do these changes tomorrow